cypress/support/pageobjects/MailinglistSignupPage.js

```diff
@@ -12,9 +12,7 @@ class MailinglistSignupPage {
 		return cy.get("#mailing-list-steps-name-current-status-field");
 	}
 	getSubmitButton() {
-		return cy.get(
-			"#edit_mailing_list_steps_name_name > .call-to-action-button"
-		);
+		return cy.get("#edit_mailing_list_steps_name_name > .call-to-action-button");
```

Comment:
I'd maybe target this by the button text? Don't feel that strongly about it though

Reply:
I agree with your comments, sure I'll change.